Content.Server/Chemistry/Components/ReagentDispenserComponent.cs

```diff
@@ -65,6 +66,21 @@ public class ReagentDispenserComponent : SharedReagentDispenserComponent, IActiv
             }
         }
 
+        /// <summary>
+        /// The solution inside the container that has been placed in this reagent dispenser.
+        /// </summary>
+        [ViewVariables]
+        private Solution? ContainedBeakerSolution
+        {
+            get
+            {
+                if (!HasBeaker)
```

Comment:
This isn't ideal. Ideally, this stuff needs to go into respective system. And there is no need for this field, since it's all handled by `SolutionContainerSystem`.

Reply:
This is fine.